apitesters/purchases.ts

```diff
@@ -124,6 +125,14 @@ async function checkConfigure() {
   const configured: boolean = await Purchases.isConfigured();
 }
 
+async function checkLogLevel() {
+  await Purchases.setLogLevel(LOG_LEVEL.DEBUG);
+  await Purchases.setLogLevel(LOG_LEVEL.VERBOSE);
+  await Purchases.setLogLevel(LOG_LEVEL.INFO);
+  await Purchases.setLogLevel(LOG_LEVEL.WARN);
+  await Purchases.setLogLevel(LOG_LEVEL.ERROR);
```

Comment:
Should we check the enum instead and just test this function with a parameter?

Reply:
Good point.